src/Public/OpenApi.ps1

```diff
@@ -3433,6 +3425,77 @@ function Select-PodeOADefinition {
 
 }
 
+<#
+.SYNOPSIS
+Renames an existing OpenAPI definition tag in Pode.
+
+.DESCRIPTION
+This function renames an existing OpenAPI definition tag to a new tag name.
+If the specified tag is the default definition tag, it updates the default tag as well.
+It ensures that the new tag name does not already exist and that the function is not used within a Select-PodeOADefinition ScriptBlock.
+
+.PARAMETER Tag
+The current tag name of the OpenAPI definition. If not specified, the default definition tag is used.
+
+.PARAMETER NewTag
+The new tag name for the OpenAPI definition. This parameter is mandatory.
+
+.EXAMPLE
+Rename-PodeOADefinitionTagName -Tag 'oldTag' -NewTag 'newTag'
+
+Rename a specific OpenAPI definition tag
+
+.EXAMPLE
+Rename-PodeOADefinitionTagName -NewTag 'newDefaultTag'
+
+Rename the default OpenAPI definition tag
+
+.NOTES
+This function will throw an error if:
+- It is used inside a Select-PodeOADefinition ScriptBlock.
+- The new tag name already exists.
+- The current tag name does not exist.
+#>
+function Rename-PodeOADefinitionTagName {
```

Comment:
The function might be better named `Rename-PodeOADefinitionTag`, dropping the "Name", as it's "DefinitionTag", "Tag", "OADefinitionTag" everywhere.

Reply:
Done